### ISSUE TITLE
Hub is recording every 5 minutes without motion being detected

### ISSUE THREAD

**Issue by kyleser** [ccrisan/motioneyeos#1708]
### Preliminary Docs

I confirm that I have read the [CONTRIBUTING](https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.

### motionEyeOS Version

I am running motionEyeOS version: 20180627

### Board Model

I am using the following board/model: ORDIOD XU4 as a hub

### Camera

I am using the following type of camera: 9 network cameras

My camera model is: raspberry pi cams.

### Network Connection

My motionEyeOS unit is connected to the network via: Ethernet

### Peripherals

I am using the following peripherals that I consider relevant to this issue: NAS

### Log Files

I consider the following log files relevant to this issue:
-attached motioneye.log for reference
[motioneye-4.log](https://github.com/ccrisan/motioneyeos/files/2555911/motioneye-4.log)

### Issue observed 
All of my cameras are recording every 5 minutes regardless if it detects motion or not.  Checking the log files showed failure to connect to the network drive however when I view my network drive I can view all the video files. I'd like to correct this so the cameras don't record every 5 minutes.

**Comment by kyleser** [comment]
Sorry, I meant to say, the file says failure to unmount.  The directory I'm using is "Security_Cams1" by the way.

**Comment by kyleser** [comment]
I have resolved this issue.  Looking at the log files it shows having issues accessing a certain file location multiple times.  I SSH'd into the hub and manually deleted all the associated and past directories but the issue remained. I then simply deleteted all the camera feeds to start over from scratch.  At this point after every addition of a camera I would have to reboot the hub before adding another one.  This somehow locks in the network drive path.  I no longer have this issue and all the cameras record correctly as expected.
